### PR TITLE
Login template should display controller's errorMessage

### PR DESCRIPTION
**Pull request by devinrhode2**
Update login template example snippit to include loginFailureMessage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,7 +171,7 @@ export default Ember.Controller.extend({
     authenticate() {
       let { identification, password } = this.getProperties('identification', 'password');
       this.get('session').authenticate('authenticator:oauth2', identification, password).catch((reason) => {
-        this.set('errorMessage', reason.error);
+        this.set('loginFailureMessage', reason.error || reason);
```
Comment (marcoow) [inline README.md]: I think `errorMessage` was just fine

File: README.md [hunk 2]
```diff
@@ -148,15 +148,12 @@ With that authenticator and a login form like
...
-    <label for="password">Password</label>
-    {{input value=password placeholder='Enter Password' class='form-control' type='password'}}
-  </div>
-  <button type="submit" class="btn btn-default">Login</button>
+  <label for="identification">Login</label>
+  {{input id='identification' placeholder='Enter Login' value=identification}}
+  <label for="password">Password</label>
+  {{input id='password' placeholder='Enter Password' type='password' value=password}}
+  <button type="submit">Login</button>
+  <p>{{loginFailureMessage}}</p>
```
Comment (marcoow) [inline README.md]: This should only render the `<p>` when there is an error actually present:

```hbs
{{#if errorMessage}}
  {{errorMessage}}
{{/if}}
```

**Comment by devinrhode2** [comment]
What do you think of my changes?

**Comment by marcoow** [comment]
👍

**Pull request merged**